rest-api/src/main/resources/paths/studies/v3_studies_studyId_oauth_vendorId_healthCode.yml

```diff
@@ -0,0 +1,22 @@
+get:
+    operationId: getHealthCodesGrantingOAuthAccess
```

Comment:
This operation name matches the one above, but is unintuitively named for what it does